lisa/base_tools/wget.py

```diff
@@ -65,3 +65,14 @@ def get(
             self.node.execute(f"chmod +x {actual_file_path}")
 
         return actual_file_path.stdout
+
+    def verify_internet_access(self) -> bool:
+        result = self.run(" microsoft.com", no_error_log=True, shell=True)
+        matched_result = self.__pattern_path.match(result.stdout)
+        if matched_result:
+            download_file_path = matched_result.group("path")
+            actual_file_path = self.node.execute(f"ls {download_file_path}", shell=True)
+            self.node.execute(f"rm {actual_file_path.stdout}", shell=True, sudo=True)
```

Comment:
If it use the shared logic, it doesn't need to use ls to find the file in this method.